.gitignore

```diff
@@ -17,3 +17,4 @@ test/version_tmp
 tmp
 vendor/bundle
 bin
+.idea
```

Comment:
You don't need to add this to `.gitignore`. You can make it "personally ignored" in `.git/info/exclude`. Thanks!